Main.py

```diff
@@ -287,15 +288,14 @@ def forward_fill(game: Game,
     return False, spoilerSave
 
 
-if __name__ == "__main__":
-    import time
-    t0 = time.perf_counter()
+def args_to_game_options(args):
+    args = args[:]
```

Comment:
I need a way to parse options in another file so I moved it into a separate function.